### PR TITLE
Automatically reset the test size between tests.

### PR DESCRIPTION
**Pull request by DaveShuckerow**
A lot of tests weren't calling `resetWindowSize` at the end of the test, which I believe was leading to the overridden window size propagating across tests.

### CODE REVIEW

File: packages/devtools_app/test/flutter/wrappers.dart [hunk 1]
```diff
@@ -45,18 +46,26 @@ Widget wrapWithControllers(
...
 }
 
-/// Sets the size of the app window under test to [windowSize].
-///
-/// This must be reset on after each test invocation that calls
-/// by using [resetWindowSize].
-Future<void> setWindowSize(Size windowSize) async {
+/// Runs a test with the size of the app window under test to [windowSize].
+@isTest
+void testWidgetsWithSize(
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/flutter/wrappers.dart]: nit: rename to `testWidgetsWithWindowSize` as `withSize` makes it seem like size is referring to the widget, not the window

File: packages/devtools_app/test/flutter/wrappers.dart [hunk 2]
```diff
@@ -45,18 +46,26 @@ Widget wrapWithControllers(
...
+    String name, Size windowSize, WidgetTesterCallback test,
+    {bool skip = false}) {
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/flutter/wrappers.dart]: trailing comma

Comment (DaveShuckerow) [reply packages/devtools_app/test/flutter/wrappers.dart]: Done.

File: packages/devtools_app/test/flutter/wrappers.dart (hunk 1 above)
Comment (DaveShuckerow) [reply packages/devtools_app/test/flutter/wrappers.dart]: Done.

Comment (DaveShuckerow) [reply packages/devtools_app/test/flutter/wrappers.dart]: done.

File: packages/devtools_app/test/flutter/wrappers.dart (hunk 2 above)
Comment (DaveShuckerow) [reply packages/devtools_app/test/flutter/wrappers.dart]: done.

**Pull request merged**